src/gg.regression.unity.bots/Runtime/Scripts/StateRecorder/ProfilerObserver.cs

```diff
@@ -0,0 +1,103 @@
+﻿using System.Collections.Generic;
+using Unity.Profiling;
+using UnityEngine;
+
+namespace StateRecorder
```

Comment:
@svolokh I didn't catch this, but this namespace is wrong.

Should be `RegressionGames.StateRecorder`... When I pulled in these changes before fixing the namespace, my project won't compile.  Please open another PR (same linear ticket is fine by me) asap to correct this

Reply:
@RG-nAmKcAz The main branch seems to compile fine on my end currently. I saw that you had performed the namespace fix in your newest PR, do you think it would be cleaner to just have this be part of the that PR's changes?